xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -107,8 +109,15 @@ extern "C" {
       TYPE_TELETEXT
     } m_streamType;
 
+    enum Codec_FEATURES
+    {
+      FEATURE_DECODE = 1
+    };
+    unsigned int m_features;
+
     char m_codecName[32];                /*!< @brief (required) name of codec according to ffmpeg */
     char m_codecInternalName[32];        /*!< @brief (optional) internal name of codec (selectionstream info) */
+    kodi::addon::CODEC_PROFILE m_codecProfile; /*!< @brief (optional) bandwidth of the stream (selectionstream info) */
```

Comment:
Prevent the use of namespaces in "C" structures. Also find it nice to have `CODEC_PROFILE` in namespace, but is then not direct "C".